### ISSUE TITLE
v9-gui: New access duration is not available in the list until refresh

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#4445]
We need to refresh the page to reload the option.

**Comment by julsemaan** [comment]
Proper fix will be done with allowed lookup

**Comment by satkunas** [comment]
After more discussion with @jrouzierinverse we've decided that there's no reliable way to automatically update a forms meta. This would require the front-end to frequently poll each API resource, and complex hooks between various store modules. This is not a direction we wish to take. 

The user will either need to refresh (F5) or click the [reset] button at the bottom of the form to reload the meta manually. This is not unreasonable since this is already a normal user-action for cache invalidation. 

TL;DR The [Reset] button at the bottom of the form will reload OPTIONS.

**Comment by lzammit** [comment]
Fixed
